02/04/2023 04:32:12PM __comp__ ERROR: 
    Name: Artiom
    Function Name: test_sumofdigits
    Description: Testing sum of number digits
    malformed node or string: None
 
Traceback (most recent call last):
  File "D:\PythonRepos\ClassLab1\test_tools\test_comp.py", line 25, in test_sumofdigits
    num = ast.literal_eval(os.getenv('NUM'))
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\deade\AppData\Local\Programs\Python\Python311\Lib\ast.py", line 110, in literal_eval
    return _convert(node_or_string)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\deade\AppData\Local\Programs\Python\Python311\Lib\ast.py", line 109, in _convert
    return _convert_signed_num(node)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\deade\AppData\Local\Programs\Python\Python311\Lib\ast.py", line 83, in _convert_signed_num
    return _convert_num(node)
           ^^^^^^^^^^^^^^^^^^
  File "C:\Users\deade\AppData\Local\Programs\Python\Python311\Lib\ast.py", line 74, in _convert_num
    _raise_malformed_node(node)
  File "C:\Users\deade\AppData\Local\Programs\Python\Python311\Lib\ast.py", line 71, in _raise_malformed_node
    raise ValueError(msg + f': {node!r}')
ValueError: malformed node or string: None
02/04/2023 04:32:36PM __comp__ ERROR: 
    Name: Artiom
    Function Name: test_sumofdigits
    Description: Testing sum of number digits
    invalid literal for int() with base 10: 'N'
 
Traceback (most recent call last):
  File "D:\PythonRepos\ClassLab1\test_tools\test_comp.py", line 26, in test_sumofdigits
    x = comp.sumofdigits(num)
        ^^^^^^^^^^^^^^^^^^^^^
  File "D:\PythonRepos\ClassLab1\tools\numbers\comp.py", line 2, in sumofdigits
    return sum(map(int, str(num)))
           ^^^^^^^^^^^^^^^^^^^^^^^
ValueError: invalid literal for int() with base 10: 'N'
02/04/2023 04:34:02PM __comp__ ERROR: 
    Name: Artiom
    Function Name: test_sumofdigits
    Description: Testing sum of number digits
    assert (6 == <map object at 0x0000020C2782EBC0>)
 +  where <map object at 0x0000020C2782EBC0> = map(int, '123')
 +    where '123' = str(123)
 
Traceback (most recent call last):
  File "D:\PythonRepos\ClassLab1\test_tools\test_comp.py", line 27, in test_sumofdigits
    assert x == map(int, str(num)) and x is not None
AssertionError: assert (6 == <map object at 0x0000020C2782EBC0>)
 +  where <map object at 0x0000020C2782EBC0> = map(int, '123')
 +    where '123' = str(123)
02/04/2023 04:34:42PM __comp__ ERROR: 
    Name: Artiom
    Function Name: test_sumofdigits
    Description: Testing sum of number digits
    invalid literal for int() with base 10: 'N'
 
Traceback (most recent call last):
  File "D:\PythonRepos\ClassLab1\test_tools\test_comp.py", line 26, in test_sumofdigits
    x = comp.sumofdigits(num)
        ^^^^^^^^^^^^^^^^^^^^^
  File "D:\PythonRepos\ClassLab1\tools\numbers\comp.py", line 2, in sumofdigits
    return sum(map(int, str(num)))
           ^^^^^^^^^^^^^^^^^^^^^^^
ValueError: invalid literal for int() with base 10: 'N'
02/04/2023 04:35:14PM __comp__ ERROR: 
    Name: Artiom
    Function Name: test_sumofdigits
    Description: Testing sum of number digits
    int() argument must be a string, a bytes-like object or a real number, not 'NoneType'
 
Traceback (most recent call last):
  File "D:\PythonRepos\ClassLab1\test_tools\test_comp.py", line 25, in test_sumofdigits
    num = int(os.getenv('NUM'))
          ^^^^^^^^^^^^^^^^^^^^^
TypeError: int() argument must be a string, a bytes-like object or a real number, not 'NoneType'
02/04/2023 04:35:39PM __comp__ ERROR: 
    Name: Artiom
    Function Name: test_sumofdigits
    Description: Testing sum of number digits
    int() argument must be a string, a bytes-like object or a real number, not 'NoneType'
 
Traceback (most recent call last):
  File "D:\PythonRepos\ClassLab1\test_tools\test_comp.py", line 25, in test_sumofdigits
    num = int(os.getenv('NUM'))
          ^^^^^^^^^^^^^^^^^^^^^
TypeError: int() argument must be a string, a bytes-like object or a real number, not 'NoneType'
02/04/2023 04:36:13PM __comp__ ERROR: 
    Name: Artiom
    Function Name: test_sumofdigits
    Description: Testing sum of number digits
    malformed node or string: None
 
Traceback (most recent call last):
  File "D:\PythonRepos\ClassLab1\test_tools\test_comp.py", line 25, in test_sumofdigits
    num = int(ast.literal_eval(os.getenv('NUM')))
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\deade\AppData\Local\Programs\Python\Python311\Lib\ast.py", line 110, in literal_eval
    return _convert(node_or_string)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\deade\AppData\Local\Programs\Python\Python311\Lib\ast.py", line 109, in _convert
    return _convert_signed_num(node)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\deade\AppData\Local\Programs\Python\Python311\Lib\ast.py", line 83, in _convert_signed_num
    return _convert_num(node)
           ^^^^^^^^^^^^^^^^^^
  File "C:\Users\deade\AppData\Local\Programs\Python\Python311\Lib\ast.py", line 74, in _convert_num
    _raise_malformed_node(node)
  File "C:\Users\deade\AppData\Local\Programs\Python\Python311\Lib\ast.py", line 71, in _raise_malformed_node
    raise ValueError(msg + f': {node!r}')
ValueError: malformed node or string: None
02/04/2023 04:39:27PM __comp__ INFO: 
    Name: Artiom
    Function Name: test_sumofdigits
    Description: Testing sum of number digits
    
Actual: 9 Expected: 9 not None
 
02/04/2023 04:44:44PM __comp__ INFO: 
    Name: Artiom
    Function Name: test_ispal
    Description: Testing checking if the number is palindrome
    
Actual: True Expected: True not None
 
02/04/2023 04:51:47PM __comp__ INFO: 
    Name: Artiom
    Function Name: test_sumofdigits
    Description: Testing sum of number digits
    
Actual: 9 Expected: 9 not None
 
02/04/2023 04:51:47PM __comp__ INFO: 
    Name: Artiom
    Function Name: test_ispal
    Description: Testing checking if the number is palindrome
    
Actual: True Expected: True not None
 
